new-frontiers-14/frontier-station-14
# Adds reinforced jug

## About the PR



This PR adds a new researchable reinforced jug. As per discussion below it is a heavy jug that is carried in two hands and carries 400u. It fits into a dispenser.

It is locked behind t2 civilian tech for the moment, as there wasn't a very ideal t1 candidate. I feel like this may be overkill and will likely revisit it in the future if some addition tech options can be opened up in t1.

The chem dispenser comes with 10 of its 20 total jugs in the form of the reinforced versions for the most "often" used reagents to provide buffer solutions for those chems and to address one of the original concerns regarding resource deprivation in the early round.

## Why / Balance


Today the issue of supplementary chem crates stored on some shuttles was raised. The general issue came down to desiring those restocks be removed in favor of vending machines, which itself was acknowledged to be an issue as vending machines have their own complications and baggage.

A simple solution would be to double the jugs present in the chem dispenser, essentially pre-loading the crate chems and removing the need for their mapping. However at present the last PR goal that I observed was to limit the initial storage capacity of the chem dispenser to 25 jugs. 

As such the option of simply (adding a jug) (with) double jug capacity seemed reasonable.

## How to test


Check research.
Check the lathe recipes.
Check the sprites, inspect fill states and general standard of quality of my horrific but well meaning artwork.

## Media


![ss+(2025-01-12+at+02 42 02)](https://github.com/user-attachments/assets/7c59704f-be97-4401-a2e7-3708f17fe69f)



## Requirements



## Breaking changes


**Changelog**


🆑 
- add: Adds reinforced jugs for expanded dispenser storage options.
